src/index.js

```diff
@@ -1,5 +1,7 @@
-// This is the JavaScript entry file - your code begins here
-// Do not delete or rename this file ********
+fetch("https://fe-apps.herokuapp.com/api/v1/gametime/1903/wheel-of-fortune/data")
+.then(response => response.json())
+.then(data => console.log(data))
+.catch(error => console.log(error));
```

Comment:
You’re doing it!!! You now have access to your game data within that second `.then()` — here’s where you might want to do all of your setup and initialization code. e.g. if you need that data in order to instantiate a new Game, you might want to do that game instantiation within this `.then()` to be sure it doesn’t try to happen before the fetch request returns